### ISSUE TITLE
Lost connectivity with all my Zigbee devices after upgrading to 0.12.0

### ISSUE THREAD

**Issue by nbackous** [WebThingsIO/zigbee-adapter#241]
Today I upgraded to version 0.12.0 of the Zigbee Add-On and I'm no longer able to connect to my Zigbee devices

![Screen Shot 2020-11-05 at 4 16 32 PM](https://user-images.githubusercontent.com/40135245/98310714-886d3500-1f82-11eb-955d-8b943a14d497.png)

![Screen Shot 2020-11-05 at 4 17 50 PM](https://user-images.githubusercontent.com/40135245/98310717-8acf8f00-1f82-11eb-9840-b54adf867748.png)

I've tried disabling/re-enabling the Add-On, removing/re-adding the Add-On and power cycling my Webthings Gateway but nothing has worked.

Here's the logs after re-adding the Add-On:

```
2020-11-05 16:40:04.407 INFO   : Fetching add-on https://github.com/WebThingsIO/zigbee-adapter/releases/download/v0.12.0/zigbee-adapter-0.12.0-linux-arm-v10.tgz as /tmp/YjR6gE/zigbee-adapter.tar.gz
2020-11-05 16:40:05.642 INFO   : Expanding add-on /tmp/YjR6gE/zigbee-adapter.tar.gz
2020-11-05 16:40:13.930 INFO   : Loading add-on: zigbee-adapter
2020-11-05 16:40:15.733 INFO   : zigbee-adapter: Opening database: /home/pi/.mozilla-iot/config/db.sqlite3
2020-11-05 16:40:16.511 INFO   : zigbee-adapter: Loading add-on zigbee-adapter from /home/pi/.mozilla-iot/addons/zigbee-adapter
2020-11-05 16:40:18.361 INFO   : zigbee-adapter: DEBUG config = ''
2020-11-05 16:40:19.218 INFO   : zigbee-adapter: this.driver = DeconzDriver {}
2020-11-05 16:40:19.221 INFO   : zigbee-adapter: DeconzDriver: Using serial port /dev/ttyACM0
2020-11-05 16:40:19.239 INFO   : zigbee-adapter: DeConz Firmware version: 264a0700
2020-11-05 16:40:19.300 INFO   : zigbee-adapter: Kicking WatchDog for 3600 seconds
2020-11-05 16:40:19.363 INFO   : zigbee-adapter: Node: undefined 628f not found
2020-11-05 16:40:19.372 INFO   : zigbee-adapter: Node: undefined 628f not found
2020-11-05 16:40:19.379 INFO   : zigbee-adapter: Node: undefined 628f not found
2020-11-05 16:40:19.386 INFO   : zigbee-adapter:          MAC Address: 00212effff053fad
2020-11-05 16:40:19.389 INFO   : zigbee-adapter:      Network PANID16: 9b8d
2020-11-05 16:40:19.390 INFO   : zigbee-adapter:       Network Addr16: 0000
2020-11-05 16:40:19.392 INFO   : zigbee-adapter:      Network PANID64: 00212effff053fad
2020-11-05 16:40:19.394 INFO   : zigbee-adapter: APS Designated Coordinator: 1
2020-11-05 16:40:19.395 INFO   : zigbee-adapter:        Scan Channels: 00008000
2020-11-05 16:40:19.397 INFO   : zigbee-adapter:          APS PANID64: 0000000000000000
2020-11-05 16:40:19.399 INFO   : zigbee-adapter:  Trust Center Addr64: 00212effff053fad
2020-11-05 16:40:19.401 INFO   : zigbee-adapter:        Security Mode: 3
2020-11-05 16:40:19.402 INFO   : zigbee-adapter:          Network Key: 944f0335924d21477ad7c99662309100
2020-11-05 16:40:19.404 INFO   : zigbee-adapter:    Operating Channel: 15
2020-11-05 16:40:19.406 INFO   : zigbee-adapter:     Protocol Version: 267
2020-11-05 16:40:19.408 INFO   : zigbee-adapter:    Network Update ID: 0
2020-11-05 16:40:19.410 INFO   : zigbee-adapter:          Permit Join: 0
2020-11-05 16:40:19.411 INFO   : zigbee-adapter:         WatchDog TTL: 3600
2020-11-05 16:40:19.412 INFO   : zigbee-adapter:              Version: 264a0700
2020-11-05 16:40:19.415 INFO   : zigbee-adapter: ZigbeeNode created: addr64: 00212effff053fad addr16: 0000
2020-11-05 16:40:19.457 INFO   : zigbee-adapter: ZigbeeNode created: addr64: 24fd5b00010b970f addr16: fed3
2020-11-05 16:40:19.458 INFO   : zigbee-adapter: ZigbeeNode created: addr64: 286d97000101d51c addr16: 03bb
2020-11-05 16:40:19.460 INFO   : zigbee-adapter: ZigbeeNode created: addr64: 84182600000c6474 addr16: d4c1
2020-11-05 16:40:19.461 INFO   : zigbee-adapter: ZigbeeNode created: addr64: 90fd9ffffef7cd98 addr16: 7b0f
2020-11-05 16:40:19.463 INFO   : zigbee-adapter: ZigbeeNode created: addr64: 7cb03eaa00a68253 addr16: 7602
2020-11-05 16:40:19.464 INFO   : zigbee-adapter: ZigbeeNode created: addr64: 7cb03eaa00a65707 addr16: ecb0
2020-11-05 16:40:19.466 INFO   : zigbee-adapter: ZigbeeNode created: addr64: 000b57fffe2ba3d4 addr16: f707
2020-11-05 16:40:19.468 INFO   : zigbee-adapter: ZigbeeNode created: addr64: 000b57fffe2da7a1 addr16: 8835
2020-11-05 16:40:19.470 INFO   : zigbee-adapter: ZigbeeNode created: addr64: 000d6ffffe1cb233 addr16: 9977
2020-11-05 16:40:19.471 INFO   : zigbee-adapter: ZigbeeNode created: addr64: 000b57fffe4ae641 addr16: 1ecf
2020-11-05 16:40:19.473 INFO   : zigbee-adapter: ZigbeeNode created: addr64: 000d6ffffe1bd006 addr16: 7897
2020-11-05 16:40:19.474 INFO   : zigbee-adapter: ZigbeeNode created: addr64: ccccccfffed6a41c addr16: 6f2c
2020-11-05 16:40:19.476 INFO   : zigbee-adapter: ZigbeeNode created: addr64: 000b57fffe2b9fc5 addr16: 6bef
2020-11-05 16:40:19.477 INFO   : zigbee-adapter: ZigbeeNode created: addr64: b0ce1814000ee1f7 addr16: 427b
2020-11-05 16:40:19.479 INFO   : zigbee-adapter: ZigbeeNode created: addr64: 24fd5b00010bf9d9 addr16: 14d1
2020-11-05 16:40:19.480 INFO   : zigbee-adapter: ZigbeeNode created: addr64: ec1bbdfffe985148 addr16: c08f
2020-11-05 16:40:19.482 INFO   : zigbee-adapter: ZigbeeNode created: addr64: 000b57fffe38ac4c addr16: 6e1d
2020-11-05 16:40:19.483 INFO   : zigbee-adapter: ZigbeeNode created: addr64: 7cb03eaa00a67daa addr16: 59af
2020-11-05 16:40:19.485 INFO   : zigbee-adapter: ZigbeeNode created: addr64: 000d6ffffe2f6221 addr16: 992d
2020-11-05 16:40:19.486 INFO   : zigbee-adapter: ZigbeeNode created: addr64: 000d6ffffe7cd8a7 addr16: 7df0
2020-11-05 16:40:19.488 INFO   : zigbee-adapter: ZigbeeNode created: addr64: 286d970001068225 addr16: 06a5
2020-11-05 16:40:19.489 INFO   : zigbee-adapter: ZigbeeNode created: addr64: 286d97000103d720 addr16: b597
2020-11-05 16:40:19.491 INFO   : zigbee-adapter: ZigbeeNode created: addr64: b0ce1814031a0cf7 addr16: 88ff
2020-11-05 16:40:19.492 INFO   : zigbee-adapter: ZigbeeNode created: addr64: b0ce1814031a079e addr16: c373
2020-11-05 16:40:19.494 INFO   : zigbee-adapter: ZigbeeNode created: addr64: 0000000000000000 addr16: 3a7f
2020-11-05 16:40:19.495 INFO   : zigbee-adapter: ZigbeeNode created: addr64: f0d1b8000001cef5 addr16: 4be3
2020-11-05 16:40:19.497 INFO   : zigbee-adapter: ZigbeeNode created: addr64: 7cb03eaa00a4b00f addr16: 95e8
2020-11-05 16:40:19.498 INFO   : zigbee-adapter: ZigbeeNode created: addr64: 286d97000101d4b2 addr16: 74c2
2020-11-05 16:40:19.500 INFO   : zigbee-adapter: ZigbeeNode created: addr64: 286d970001076fa1 addr16: e76f
2020-11-05 16:40:19.501 INFO   : zigbee-adapter: ZigbeeNode created: addr64: 24fd5b0001089718 addr16: 8197
2020-11-05 16:40:19.503 INFO   : zigbee-adapter: ZigbeeNode created: addr64: 286d970001060616 addr16: 877c
2020-11-05 16:40:19.504 INFO   : zigbee-adapter: ZigbeeNode created: addr64: 7cb03eaa00a656cb addr16: 0b5e
2020-11-05 16:40:19.506 INFO   : zigbee-adapter: ZigbeeNode created: addr64: 84182600000c65f0 addr16: 37d7
2020-11-05 16:40:19.507 INFO   : zigbee-adapter: ZigbeeNode created: addr64: 7cb03eaa00a655ed addr16: 5d72
2020-11-05 16:40:19.509 INFO   : zigbee-adapter: ZigbeeNode created: addr64: 7cb03eaa00a6716c addr16: 2d54
2020-11-05 16:40:19.510 INFO   : zigbee-adapter: ZigbeeNode created: addr64: 286d9700010c7347 addr16: 5edb
2020-11-05 16:40:19.512 INFO   : zigbee-adapter: ZigbeeNode created: addr64: 7cb03eaa00a632fe addr16: 3f9e
2020-11-05 16:40:19.513 INFO   : zigbee-adapter: ZigbeeNode created: addr64: 00158d00023899b1 addr16: 78e0
2020-11-05 16:40:19.516 INFO   : zigbee-adapter: ZigbeeNode created: addr64: 24fd5b000104fded addr16: a5ce
2020-11-05 16:40:19.517 INFO   : zigbee-adapter: ZigbeeNode created: addr64: 24fd5b000104fe71 addr16: 7d0f
2020-11-05 16:40:19.522 INFO   : getValue for property humidity for: zb-00158d00023899b1-temperature returning 0
2020-11-05 16:40:19.540 INFO   : getValue for property temperature for: zb-00158d00023899b1-temperature returning 0
2020-11-05 16:40:19.542 INFO   : getValue for property pressure for: zb-00158d00023899b1-temperature returning 0
2020-11-05 16:40:19.561 INFO   : zigbee-adapter: handleNetworkAddressResponse: Skipping: 00155f00b44c963d due to unknown addr64
2020-11-05 16:40:19.566 INFO   : zigbee-adapter: handleNetworkAddressResponse: Skipping: 00155f00b44c963d due to unknown addr64
2020-11-05 16:40:19.673 INFO   : zigbee-adapter: handleNetworkAddressResponse: Skipping: 00155f00b44c963d due to unknown addr64
2020-11-05 16:40:19.692 INFO   : zigbee-adapter: ----- Scan Starting -----
2020-11-05 16:40:19.745 INFO   : zigbee-adapter: Processing endpoint 0 = 1
2020-11-05 16:40:19.747 ERROR  : zigbee-adapter: Error handling frame_object
2020-11-05 16:40:19.748 ERROR  : zigbee-adapter: ReferenceError: genScenesOutputEndpoints is not defined
2020-11-05 16:40:19.750 ERROR  : zigbee-adapter:     at ZigbeeClassifier.initMultiLevelButtons (/home/pi/.mozilla-iot/addons/zigbee-adapter/zb-classifier.js:2080:45)
2020-11-05 16:40:19.751 ERROR  : zigbee-adapter:     at ZigbeeClassifier.classifyInternal (/home/pi/.mozilla-iot/addons/zigbee-adapter/zb-classifier.js:1809:12)
2020-11-05 16:40:19.752 ERROR  : zigbee-adapter:     at ZigbeeClassifier.classify (/home/pi/.mozilla-iot/addons/zigbee-adapter/zb-classifier.js:1846:10)
2020-11-05 16:40:19.753 ERROR  : zigbee-adapter:     at ZigbeeNode.classify (/home/pi/.mozilla-iot/addons/zigbee-adapter/zb-node.js:239:24)
2020-11-05 16:40:19.755 ERROR  : zigbee-adapter:     at ZigbeeAdapter.handleDeviceAdded (/home/pi/.mozilla-iot/addons/zigbee-adapter/zb-adapter.js:1350:12)
2020-11-05 16:40:19.756 ERROR  : zigbee-adapter:     at ZigbeeAdapter.addIfReady (/home/pi/.mozilla-iot/addons/zigbee-adapter/zb-adapter.js:1334:10)
2020-11-05 16:40:19.757 ERROR  : zigbee-adapter:     at ZigbeeAdapter.populateNodeInfoEndpoints (/home/pi/.mozilla-iot/addons/zigbee-adapter/zb-adapter.js:1608:10)
2020-11-05 16:40:19.758 ERROR  : zigbee-adapter:     at ZigbeeAdapter.populateNodeInfo (/home/pi/.mozilla-iot/addons/zigbee-adapter/zb-adapter.js:1375:12)
2020-11-05 16:40:19.759 ERROR  : zigbee-adapter:     at ZigbeeAdapter.scanNode (/home/pi/.mozilla-iot/addons/zigbee-adapter/zb-adapter.js:159:10)
2020-11-05 16:40:19.761 ERROR  : zigbee-adapter:     at ZigbeeAdapter.enumerateNextNode (/home/pi/.mozilla-iot/addons/zigbee-adapter/zb-adapter.js:403:12)
2020-11-05 16:40:19.764 ERROR  : zigbee-adapter: { response: true,
2020-11-05 16:40:19.765 ERROR  : zigbee-adapter:   type: 23,
2020-11-05 16:40:19.767 ERROR  : zigbee-adapter:   seqNum: 59,
2020-11-05 16:40:19.768 ERROR  : zigbee-adapter:   status: 0,
2020-11-05 16:40:19.769 ERROR  : zigbee-adapter:   payloadLen: 72,
2020-11-05 16:40:19.770 ERROR  : zigbee-adapter:   deviceState: 38,
2020-11-05 16:40:19.771 ERROR  : zigbee-adapter:   networkState: 2,
2020-11-05 16:40:19.773 ERROR  : zigbee-adapter:   dataConfirm: true,
2020-11-05 16:40:19.774 ERROR  : zigbee-adapter:   dataIndication: false,
2020-11-05 16:40:19.775 ERROR  : zigbee-adapter:   configChanged: false,
2020-11-05 16:40:19.776 ERROR  : zigbee-adapter:   dataRequest: true,
2020-11-05 16:40:19.777 ERROR  : zigbee-adapter:   dstAddrMode: 2,
2020-11-05 16:40:19.778 ERROR  : zigbee-adapter:   destination16: '0000',
2020-11-05 16:40:19.779 ERROR  : zigbee-adapter:   destinationEndpoint: '00',
2020-11-05 16:40:19.781 ERROR  : zigbee-adapter:   srcAddrMode: 2,
2020-11-05 16:40:19.782 ERROR  : zigbee-adapter:   remote16: '0000',
2020-11-05 16:40:19.783 ERROR  : zigbee-adapter:   sourceEndpoint: '00',
2020-11-05 16:40:19.784 ERROR  : zigbee-adapter:   profileId: '0000',
2020-11-05 16:40:19.785 ERROR  : zigbee-adapter:   clusterId: '8031',
2020-11-05 16:40:19.786 ERROR  : zigbee-adapter:   data:
2020-11-05 16:40:19.787 ERROR  : zigbee-adapter:    <Buffer 0d 00 12 10 02 ad 3f 05 ff ff 2e 21 00 f0 65 0c 00 00 26 18 84 d7 37 25 01 01 9b ad 3f 05 ff ff 2e 21 00 00 00 00 00 00 00 00 00 e0 78 25 01 01 cf>,
2020-11-05 16:40:19.789 ERROR  : zigbee-adapter:   lqi: 255,
2020-11-05 16:40:19.790 ERROR  : zigbee-adapter:   rssi: -49,
2020-11-05 16:40:19.791 ERROR  : zigbee-adapter:   zdoSeq: 13,
2020-11-05 16:40:19.792 ERROR  : zigbee-adapter:   numEntries: 18,
2020-11-05 16:40:19.793 ERROR  : zigbee-adapter:   startIndex: 16,
2020-11-05 16:40:19.794 ERROR  : zigbee-adapter:   numEntriesThisResponse: 2,
2020-11-05 16:40:19.795 ERROR  : zigbee-adapter:   neighbors:
2020-11-05 16:40:19.801 ERROR  : zigbee-adapter:    [ { panId: '00212effff053fad',
2020-11-05 16:40:19.802 ERROR  : zigbee-adapter:        addr64: '84182600000c65f0',
2020-11-05 16:40:19.803 ERROR  : zigbee-adapter:        addr16: '37d7',
2020-11-05 16:40:19.804 ERROR  : zigbee-adapter:        deviceType: 1,
2020-11-05 16:40:19.805 ERROR  : zigbee-adapter:        rxOnWhenIdle: 1,
2020-11-05 16:40:19.807 ERROR  : zigbee-adapter:        relationship: 2,
2020-11-05 16:40:19.808 ERROR  : zigbee-adapter:        permitJoining: 1,
2020-11-05 16:40:19.809 ERROR  : zigbee-adapter:        depth: 1,
2020-11-05 16:40:19.810 ERROR  : zigbee-adapter:        lqi: 155 },
2020-11-05 16:40:19.811 ERROR  : zigbee-adapter:      { panId: '00212effff053fad',
2020-11-05 16:40:19.812 ERROR  : zigbee-adapter:        addr64: '00158d00023899b1',
2020-11-05 16:40:19.814 ERROR  : zigbee-adapter:        addr16: '78e0',
2020-11-05 16:40:19.815 ERROR  : zigbee-adapter:        deviceType: 1,
2020-11-05 16:40:19.816 ERROR  : zigbee-adapter:        rxOnWhenIdle: 1,
2020-11-05 16:40:19.817 ERROR  : zigbee-adapter:        relationship: 2,
2020-11-05 16:40:19.818 ERROR  : zigbee-adapter:        permitJoining: 1,
2020-11-05 16:40:19.820 ERROR  : zigbee-adapter:        depth: 1,
2020-11-05 16:40:19.821 ERROR  : zigbee-adapter:        lqi: 207 } ],
2020-11-05 16:40:19.822 ERROR  : zigbee-adapter:   remote64: '00212effff053fad' }
2020-11-05 16:40:23.231 INFO   : zigbee-adapter: Node: undefined 628f not found
2020-11-05 16:40:23.289 INFO   : zigbee-adapter: handleNetworkAddressResponse: Skipping: 00155f00b44c963d due to unknown addr64
```

**Comment by IndrekHaav** [comment]
Same problem here - Zigbee add-on was automatically updated and Zigbee devices stopped working. Gateway version 0.12.0, running on a RPi 3 with Conbee II.

Logs:

```
2020-11-06 09:22:11.393 INFO   : Checking for add-on updates...
2020-11-06 09:22:12.039 INFO   : Fetching add-on https://github.com/WebThingsIO/zigbee-adapter/releases/download/v0.12.0/zigbee-adapter-0.12.0-linux-arm-v10.tgz as /tmp/25HD3b/zigbee-adapter.tar.gz
2020-11-06 09:22:14.074 INFO   : Expanding add-on /tmp/25HD3b/zigbee-adapter.tar.gz
2020-11-06 09:22:15.842 INFO   : Unloading ZigbeeAdapter
2020-11-06 09:22:15.847 INFO   : zigbee-adapter: Adapter: ZigbeeAdapter unloaded
2020-11-06 09:22:21.227 INFO   : Loading add-on: zigbee-adapter
2020-11-06 09:22:21.230 INFO   : Finished updating add-ons
2020-11-06 09:22:22.935 INFO   : zigbee-adapter: Opening database: /home/pi/.mozilla-iot/config/db.sqlite3
2020-11-06 09:22:23.644 INFO   : zigbee-adapter: Loading add-on zigbee-adapter from /home/pi/.mozilla-iot/addons/zigbee-adapter
2020-11-06 09:22:25.418 INFO   : zigbee-adapter: DEBUG config = ''
2020-11-06 09:22:26.404 INFO   : zigbee-adapter: this.driver = DeconzDriver {}
2020-11-06 09:22:26.408 INFO   : zigbee-adapter: DeconzDriver: Using serial port /dev/ttyACM0
2020-11-06 09:22:26.427 INFO   : zigbee-adapter: DeConz Firmware version: 26490700
2020-11-06 09:22:26.487 INFO   : zigbee-adapter: Kicking WatchDog for 3600 seconds
2020-11-06 09:22:26.527 INFO   : zigbee-adapter:          MAC Address: 00212effff0426f5
2020-11-06 09:22:26.530 INFO   : zigbee-adapter:      Network PANID16: 6e7c
2020-11-06 09:22:26.532 INFO   : zigbee-adapter:       Network Addr16: 0000
2020-11-06 09:22:26.534 INFO   : zigbee-adapter:      Network PANID64: 00212effff0426f5
2020-11-06 09:22:26.536 INFO   : zigbee-adapter: APS Designated Coordinator: 1
2020-11-06 09:22:26.538 INFO   : zigbee-adapter:        Scan Channels: 00008000
2020-11-06 09:22:26.540 INFO   : zigbee-adapter:          APS PANID64: 0000000000000000
2020-11-06 09:22:26.543 INFO   : zigbee-adapter:  Trust Center Addr64: 00212effff0426f5
2020-11-06 09:22:26.545 INFO   : zigbee-adapter:        Security Mode: 3
2020-11-06 09:22:26.547 INFO   : zigbee-adapter:          Network Key: 2d0ff39b21e6e739756dfb5ee9236f00
2020-11-06 09:22:26.549 INFO   : zigbee-adapter:    Operating Channel: 15
2020-11-06 09:22:26.551 INFO   : zigbee-adapter:     Protocol Version: 267
2020-11-06 09:22:26.553 INFO   : zigbee-adapter:    Network Update ID: 0
2020-11-06 09:22:26.555 INFO   : zigbee-adapter:          Permit Join: 0
2020-11-06 09:22:26.557 INFO   : zigbee-adapter:         WatchDog TTL: 3600
2020-11-06 09:22:26.559 INFO   : zigbee-adapter:              Version: 26490700
2020-11-06 09:22:26.562 INFO   : zigbee-adapter: ZigbeeNode created: addr64: 00212effff0426f5 addr16: 0000
2020-11-06 09:22:26.564 INFO   : zigbee-adapter: ZigbeeNode created: addr64: 90fd9ffffe64d740 addr16: 3a54
2020-11-06 09:22:26.566 INFO   : zigbee-adapter: ZigbeeNode created: addr64: 7cb03eaa0a00ea29 addr16: 9a0f
2020-11-06 09:22:26.569 INFO   : zigbee-adapter: ZigbeeNode created: addr64: f0d1b80000125deb addr16: c3d3
2020-11-06 09:22:26.571 INFO   : zigbee-adapter: ZigbeeNode created: addr64: 90fd9ffffe326f4a addr16: 9e7d
2020-11-06 09:22:26.573 INFO   : zigbee-adapter: ZigbeeNode created: addr64: 00158d0003f0d58d addr16: 6e02
2020-11-06 09:22:26.589 INFO   : zigbee-adapter: ZigbeeNode created: addr64: 00158d000411bd79 addr16: 5bac
2020-11-06 09:22:26.621 INFO   : zigbee-adapter: ZigbeeNode created: addr64: 00158d000411bbb3 addr16: 34cd
2020-11-06 09:22:26.634 INFO   : zigbee-adapter: ZigbeeNode created: addr64: 0000000000000000 addr16: 34cd
2020-11-06 09:22:26.636 INFO   : zigbee-adapter: ZigbeeNode created: addr64: 00158d00041ef864 addr16: 5891
2020-11-06 09:22:26.674 INFO   : zigbee-adapter: ----- Scan Starting -----
2020-11-06 09:22:26.696 INFO   : getValue for property on for: zb-90fd9ffffe64d740-Node returning true
2020-11-06 09:22:26.700 INFO   : zigbee-adapter: genOnOffEndpoints = [ 3 ]
2020-11-06 09:22:26.702 INFO   : zigbee-adapter: Processing endpoing 0 = 3
2020-11-06 09:22:26.717 INFO   : zigbee-adapter: genOnOffEndpoints = [ 1 ]
2020-11-06 09:22:26.720 INFO   : zigbee-adapter: Processing endpoing 0 = 1
2020-11-06 09:22:26.725 INFO   : zigbee-adapter: Processing endpoint 0 = 1
2020-11-06 09:22:26.727 ERROR  : zigbee-adapter: Error handling frame_object
2020-11-06 09:22:26.729 ERROR  : zigbee-adapter: ReferenceError: genScenesOutputEndpoints is not defined
2020-11-06 09:22:26.731 ERROR  : zigbee-adapter:     at ZigbeeClassifier.initMultiLevelButtons (/home/pi/.mozilla-iot/addons/zigbee-adapter/zb-classifier.js:2080:45)
2020-11-06 09:22:26.732 ERROR  : zigbee-adapter:     at ZigbeeClassifier.classifyInternal (/home/pi/.mozilla-iot/addons/zigbee-adapter/zb-classifier.js:1809:12)
2020-11-06 09:22:26.734 ERROR  : zigbee-adapter:     at ZigbeeClassifier.classify (/home/pi/.mozilla-iot/addons/zigbee-adapter/zb-classifier.js:1846:10)
2020-11-06 09:22:26.735 ERROR  : zigbee-adapter:     at ZigbeeNode.classify (/home/pi/.mozilla-iot/addons/zigbee-adapter/zb-node.js:239:24)
2020-11-06 09:22:26.736 ERROR  : zigbee-adapter:     at ZigbeeAdapter.handleDeviceAdded (/home/pi/.mozilla-iot/addons/zigbee-adapter/zb-adapter.js:1350:12)
2020-11-06 09:22:26.737 ERROR  : zigbee-adapter:     at ZigbeeAdapter.addIfReady (/home/pi/.mozilla-iot/addons/zigbee-adapter/zb-adapter.js:1334:10)
2020-11-06 09:22:26.737 ERROR  : zigbee-adapter:     at ZigbeeAdapter.populateNodeInfoEndpoints (/home/pi/.mozilla-iot/addons/zigbee-adapter/zb-adapter.js:1608:10)
2020-11-06 09:22:26.738 ERROR  : zigbee-adapter:     at ZigbeeAdapter.populateNodeInfo (/home/pi/.mozilla-iot/addons/zigbee-adapter/zb-adapter.js:1375:12)
2020-11-06 09:22:26.739 ERROR  : zigbee-adapter:     at ZigbeeAdapter.scanNode (/home/pi/.mozilla-iot/addons/zigbee-adapter/zb-adapter.js:159:10)
2020-11-06 09:22:26.740 ERROR  : zigbee-adapter:     at ZigbeeAdapter.enumerateNextNode (/home/pi/.mozilla-iot/addons/zigbee-adapter/zb-adapter.js:403:12)
2020-11-06 09:22:26.742 ERROR  : zigbee-adapter: { response: true,
2020-11-06 09:22:26.743 ERROR  : zigbee-adapter:   type: 23,
2020-11-06 09:22:26.744 ERROR  : zigbee-adapter:   seqNum: 26,
2020-11-06 09:22:26.745 ERROR  : zigbee-adapter:   status: 0,
2020-11-06 09:22:26.746 ERROR  : zigbee-adapter:   payloadLen: 50,
2020-11-06 09:22:26.747 ERROR  : zigbee-adapter:   deviceState: 38,
2020-11-06 09:22:26.747 ERROR  : zigbee-adapter:   networkState: 2,
2020-11-06 09:22:26.748 ERROR  : zigbee-adapter:   dataConfirm: true,
2020-11-06 09:22:26.750 ERROR  : zigbee-adapter:   dataIndication: false,
2020-11-06 09:22:26.751 ERROR  : zigbee-adapter:   configChanged: false,
2020-11-06 09:22:26.751 ERROR  : zigbee-adapter:   dataRequest: true,
2020-11-06 09:22:26.753 ERROR  : zigbee-adapter:   dstAddrMode: 2,
2020-11-06 09:22:26.754 ERROR  : zigbee-adapter:   destination16: '0000',
2020-11-06 09:22:26.755 ERROR  : zigbee-adapter:   destinationEndpoint: '00',
2020-11-06 09:22:26.756 ERROR  : zigbee-adapter:   srcAddrMode: 2,
2020-11-06 09:22:26.757 ERROR  : zigbee-adapter:   remote16: '0000',
2020-11-06 09:22:26.757 ERROR  : zigbee-adapter:   sourceEndpoint: '00',
2020-11-06 09:22:26.759 ERROR  : zigbee-adapter:   profileId: '0000',
2020-11-06 09:22:26.760 ERROR  : zigbee-adapter:   clusterId: '8031',
2020-11-06 09:22:26.761 ERROR  : zigbee-adapter:   data:
2020-11-06 09:22:26.762 ERROR  : zigbee-adapter:    <Buffer 03 00 03 02 01 f5 26 04 ff ff 2e 21 00 29 ea 00 0a aa 3e b0 7c 0f 9a 25 01 01 cf>,
2020-11-06 09:22:26.763 ERROR  : zigbee-adapter:   lqi: 136,
2020-11-06 09:22:26.764 ERROR  : zigbee-adapter:   rssi: 51,
2020-11-06 09:22:26.765 ERROR  : zigbee-adapter:   zdoSeq: 3,
2020-11-06 09:22:26.766 ERROR  : zigbee-adapter:   numEntries: 3,
2020-11-06 09:22:26.767 ERROR  : zigbee-adapter:   startIndex: 2,
2020-11-06 09:22:26.768 ERROR  : zigbee-adapter:   numEntriesThisResponse: 1,
2020-11-06 09:22:26.768 ERROR  : zigbee-adapter:   neighbors:
2020-11-06 09:22:26.769 ERROR  : zigbee-adapter:    [ { panId: '00212effff0426f5',
2020-11-06 09:22:26.770 ERROR  : zigbee-adapter:        addr64: '7cb03eaa0a00ea29',
2020-11-06 09:22:26.771 ERROR  : zigbee-adapter:        addr16: '9a0f',
2020-11-06 09:22:26.772 ERROR  : zigbee-adapter:        deviceType: 1,
2020-11-06 09:22:26.773 ERROR  : zigbee-adapter:        rxOnWhenIdle: 1,
2020-11-06 09:22:26.774 ERROR  : zigbee-adapter:        relationship: 2,
2020-11-06 09:22:26.775 ERROR  : zigbee-adapter:        permitJoining: 1,
2020-11-06 09:22:26.776 ERROR  : zigbee-adapter:        depth: 1,
2020-11-06 09:22:26.777 ERROR  : zigbee-adapter:        lqi: 207 } ],
2020-11-06 09:22:26.778 ERROR  : zigbee-adapter:   remote64: '00212effff0426f5' }
```

**Comment by IndrekHaav** [comment]
I've submitted a PR that should fix the issue. I tested it on my RPi - edited `zb-classifier.js`, updated `SHA256SUMS` accordingly and disabled then enabled the Zigbee add-on. All Zigbee devices started working again.

**Comment by mrstegeman** [comment]
Should be fixed in v0.12.1.
